sapi/fpm/fpm/fpm_env.c

```diff
@@ -124,10 +124,11 @@ void fpm_env_setproctitle(char *title) /* {{{ */
 	setproctitle("%s", title);
 #else
 #ifdef __linux__
-	if (fpm_env_argv != NULL && fpm_env_argv_len > strlen(SETPROCTITLE_PREFIX) + 3) {
+	static size_t prefixlen = strlen(SETPROCTITLE_PREFIX);
```

Comment:
Any reason to make this a global (static) variable?

Note that `SETPROCTITLE_PREFIX` is a string literal, so the repeated computation doesn't matter for performance (only for coding style).

Reply:
> Any reason to make this a global (static) variable?
nope indeed.